attendance/views.py

```diff
@@ -101,10 +95,16 @@ def compile_daily_attendance_for_course(course_id):
 
 
 def get_average_attendance_from_list(list):
-    return sum(
-        attendance_object.presence == "Present" or attendance_object.presence == "Late"
-        for attendance_object in list
-    ) / len(list)
+    return (
+        sum(
+            attendance_object.presence == "Present"
+            or attendance_object.presence == "Late"
+            for attendance_object in list
+        )
+        / len(list)
+        if len(list) > 0
+        else 0
+    )
```

Comment:
This could be simplified with [statistics.mean](https://docs.python.org/3/library/statistics.html#statistics.mean), although you do still have to deal with the degenerate case either way

Reply:
Changed it in a52778685ea63cb9b3df8f6c3db475d64e5779e7